validator/rpc/wallet.go

```diff
@@ -165,6 +133,78 @@ func (s *Server) WalletConfig(ctx context.Context, _ *empty.Empty) (*pb.WalletRe
 	}, nil
 }
 
+// RecoverWallet via an API request, allowing a user to recover a derived.
+// Generate the seed from the mnemonic + language + 25th passphrase(optional).
+// Create N validator keystores from the seed specified by req.NumAccounts.
+// Set the wallet password to req.WalletPassword, then create the wallet from
+// the provided Mnemonic and return CreateWalletResponse,nil if nothing went wrong
+func (s *Server) RecoverWallet(ctx context.Context, req *pb.RecoverWalletRequest) (*pb.CreateWalletResponse, error) {
+	numAccounts := int(req.NumAccounts)
+	if numAccounts < 1 {
```

Comment:
```suggestion
	if numAccounts == 0 {
```
for a clearer validation condition.